migrations/versions/307_delete_dm_datetime.py

```diff
@@ -0,0 +1,43 @@
+"""
+
+Revision ID: 307
```

Comment:
The revision ID needs to match the file name here and on line 12. (Because we compare the latest migration file name to the revision ID in the database to see if we need to run test the migrations when deploying)

Reply:
Thanks, believe I've addressed your comment